src/kedro_databricks/bundle.py

```diff
@@ -258,3 +270,17 @@ def save_bundled_resources(
                 yaml.dump(
                     resource, f, default_flow_style=False, indent=4, sort_keys=False
                 )
+
+
+def _join_runtime_parameters(runtime_params: list[str]):
+    return ",".join(["=" .join(pair) for pair in _batched(runtime_params, 2)])
+
+
+def _batched(iterable, n):
```

Comment:
Not a huge fan of this. Since it's parsing key value pairs, I'd like to see a deterministic implementation

Reply:
I'm not sure to understand: runtime_params is a list that was generated by splitting a string in line 52.

Reply:
I moved the formatting in the `__init__` method to make it clearer.

Reply:
The discussion made me realise that it would be much simpler to let the user pass the arguments in the right format.

See next commit.